tests/oid/test_oid_spec.lua

```diff
@@ -1,150 +1,152 @@
 -- Tests for luagit2's oid module functions
 
 describe("oid methods test", function()
-
-	local lib = require("luagit2")
-	local Obj_Id
-	local Obj_Id_String = "1385f264afb75a56a5bec74243be9b367ba4ca08" -- Object ID of Fixtures/testrepo2/README
-	local Obj_Id_semi_zero_A, Obj_Id_semi_zero_B
-	local Obj_Id_semi_zero_string_A = "1385f264afb75a56a5b000000000000000000000"
-	local Obj_Id_semi_zero_string_B = "0000000000000006a5bec74243be9b367ba4ca08"
-	local Obj_Id_2
-	local Obj_Id_String_2 = "fa49b077972391ad58037050f2a75f74e3671e92" -- Object ID of Fixtures/testrepo2/new.txt
-	local Zero_Obj_Id
-	local Zero_Obj_Id_String = "0000000000000000000000000000000000000000"
-	local Extra_Object -- A Extra variable to demonstrate `Dis-Similar` Object Creation
-
+    
+    local luagit2 = require("luagit2")
+    local obj_id
```

Comment:
I think these ones are better declared inside the specifc tests than here at the top since it is not initialised here.

Reply:
These variables get initialized in `before_each()`. These are used in many of the tests therefore I listed them here, separately. 

There is no other method to have them directly initialized, not without few extra steps that I used in `before_each()`.

Reply:
Oh yes, I missed that. Ignore these comments then

Reply:
alright :smile: